src/Raven.Server/Documents/Indexes/IndexDefinitionBase.cs

```diff
@@ -155,6 +174,13 @@ static IndexDefinitionBase()
             }
         }
 
+        public override long Version => _indexVersion;
+
+        internal override void Reset()
+        {
+            _indexVersion = IndexVersion.CurrentVersion;
```

Comment:
What would persist this change after this?

Reply:
This is used during index Reset only and during that operation we are persisting whole definition again because index is being recreated